### ISSUE TITLE
Blank when scrolling down on subscriptions page

### ISSUE THREAD

**Issue by ThomasAlban** [qnblackcat/uYouPlus#1332]
On the youtube subscriptions page, the page goes blank when scrolling down momentarily. The problem only seems to occur on the subscriptions page, and not on the home page etc.

https://github.com/qnblackcat/uYouPlus/assets/98399119/f28550f4-5379-4afd-afeb-62563dffb80e

**Comment by Nesci28** [comment]
Same thing here but on home screen

https://github.com/qnblackcat/uYouPlus/assets/33325461/6722d21e-90ce-433d-9844-15762442a660

**Comment by quoctrile8** [comment]
Turn off the "Hide Community Posts" option in uYouEnhanced should fix this problem!
[image](https://github.com/qnblackcat/uYouPlus/assets/140263905/c9083131-ecc3-4132-96e7-4fe2bbd697a1)

**Comment by therealFoxster** [comment]
@ThomasAlban I think yours is caused by uYou’s “Hide Shorts Cells” option. I’ve had this glitch too when I enabled that option.

**Comment by therealFoxster** [comment]
@Nesci28 Are you using any options to hide UI elements? I see you got Shorts there so it must not be hide Shorts?

**Comment by akhilyeddula** [comment]
I am going to chime in because I have a very similar issue. The Home Screen will not properly show as shown in the video. This is a fresh install too. I uninstalled and reinstalled the app completely. 15 Pro Max, iOS 17.3.1, non-jailbroken.

https://github.com/qnblackcat/uYouPlus/assets/44342102/da308389-dd71-4180-8e71-702816a55d07

**Comment by therealFoxster** [comment]
@akhilyeddula I'm assuming you're on the latest version? It's weird cause I used to get these random blanks while scrolling my feeds too but I haven't seen them in a bit

**Comment by ThomasAlban** [comment]
> @ThomasAlban I think yours is caused by uYou’s “Hide Shorts Cells” option. I’ve had this glitch too when I enabled that option.

yeah you're right it is caused by this. Do you know when this will be fixed as hiding shorts cells is quite useful?

**Comment by Nesci28** [comment]
> @Nesci28 Are you using any options to hide UI elements? I see you got Shorts there so it must not be hide Shorts?

no, not that i know off.
fresh installation, latest version.

**Comment by akhilyeddula** [comment]
> @akhilyeddula I'm assuming you're on the latest version? It's weird cause I used to get these random blanks while scrolling my feeds too but I haven't seen them in a bit

Yep, installed through AltStore. I wonder whether it's actually related to this app...I have tried 2 other YouTube tweaks (uYou+, uYouEnhanced) and they yield the same result. However, I will say that the most recent version of this one didn't let me sign in. It said that this app couldn't be trusted or something along those lines.

**Comment by adham-elaraby** [comment]
Its not because of the hide shorts cells, its because of the disable add option in uyou

**Comment by zbcochran** [comment]
I just tried the same thing; everything works fine when logged out of my account, but once logged back in, the blanks appear again.

**Comment by kyutori** [comment]
Me too.
When i use 18.14.1 3.0 version, There is not blank
After i updated 19.06.2 3.0.3 version, blank appear.
lately i updated 19.09.4 3.0.3 version, It is not fixed

When i logout youtube account in uyou+, It works right
